cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -179,7 +180,8 @@ public ApplicationManager deployApplication(Id.Application appId,
   }
 
   @Override
-  public ApplicationManager getApplicationManager(Id.Application appId) {
+  public ApplicationManager getApplicationManager(ApplicationId applicationId) {
+    Id.Application appId = Id.Application.from(applicationId.getNamespace(), applicationId.getApplication());
     return new RemoteApplicationManager(appId, clientConfig, restClient);
```

Comment:
you can just do ``return new RemoteAppicationManager(applicationId.toId(), clientConfig, restClient)``